### DISCUSSION TITLE
ZuluSCSI RP2040 rev 2023a - how to connect CD-ROM eject switches?

### DISCUSSION THREAD

**Discussion by csulaguy** [ZuluSCSI/ZuluSCSI-firmware#464] [Q&A] [answered]
Also got this other new revision. I cannot find any photos or documentation of this board, and again, the pinouts are different, so I'm not sure where I can connect eject switches to this board, or again, if the features have been deprecated without notice.

**Comment by tenox7** [comment]
This is how I wired mine if this helps:

![IMG_5258](https://github.com/user-attachments/assets/e9d7c399-0804-4382-b37a-93d8e6ca785d)

**Comment by cathoderaydude** [comment, marked as answer]
tenox7, thank you for your photo, helped me figure it out. Here's another pic with all three pins labeled for clarity.

For anyone happening across this thread, to save you ten minutes: yes, you can just short these with a paperclip or pair of pliers. To see if it's actually switching and to which image, hook up a micro-USB cable to the USB port and connect to the virtual serial device that appears.

![image](https://github.com/user-attachments/assets/fd898144-9f22-4e94-a110-28ceff4e37c4)
